contentcuration/kolibri_content/models.py

```diff
@@ -0,0 +1,300 @@
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from jsonfield import JSONField
+from kolibri.core.fields import DateTimeTzField
+from kolibri.content import legacy_models
+from le_utils.constants import content_kinds, file_formats, format_presets
+from le_utils.constants.languages import LANGUAGE_DIRECTIONS
+from mptt.models import MPTTModel, TreeForeignKey
+
+
+class License(legacy_models.License):
```

Comment:
What's in `legacy_models`, and is there a better way to use it, since it looks like it's only used for the `License` model? Maybe we can put it in the studio repo instead of on Kolibri itself.

Reply:
When we abstract the base content classes in Kolbri we can store the legacy fields on other models in here.

I prefer to put it in Kolibri to avoid the maintenance burden on Studio of having to track these things.